python/lsst/ts/fbs/utils/maintel/sv_config.py

```diff
@@ -76,10 +77,7 @@ def survey_footprint(
     # Reduced to the following on 2025-08-10 following bad weather
     # and a redirection of time towards image quality improvements.
     wide_area = np.where(
-        (
-            (np.abs(allsky["eclip_lat"]) < 5)
-            & ((allsky["eclip_lon"] > 285) | (allsky["eclip_lon"] < -5))
-        ),
+        ((np.abs(allsky["eclip_lat"]) < 5) & (allsky["eclip_lon"] > 285)),
```

Comment:
Did you accidentally revert a change here? Or was this an intentional increase as part of the switch to the constant footprint? It looks like we removed the eclip_lon < -5 deg from the constraints. If this was intentional, could you please add it to the notes in the version history?